### PR TITLE
Mem fix

### PR DESCRIPTION
**Pull request by vext01**
Fix memory efficiency of plot script, and make the script resilient to missing instr data files.

The memory fix basically just makes it so that only the instrumentation data we need is loaded (a ChartData is for one pexec only now, not all pexecs for a given key) and that only one instr file is loaded in at a time.

memory consumption for the data we have typically bounces between around 4 and 16GB, whereas before it would outright crash.

Tested with and without -b, with and without --onepage, and with missing instr files. Seems ok.

OK?

### CODE REVIEW

File: bin/plot_krun_results [hunk 1]
```diff
@@ -122,17 +122,28 @@ EXPORT_SIZE_INCHES = [12, 10]
...
-        parser = INSTRUMENTATION_PARSERS[vm](merge_instr_data(pexec_data))
-        data_dictionary['instr_data'][key][machine] = parser.chart_data
+        for pexec_idx in pexec_idxs:
+            file_ = os.path.join(instr_dir, "%s__%s__%s__%s.json.bz2" %
+                                 (bench, vm, variant, pexec_idx))
+            print("Loading: %s" % file_)
+            try:
+                js = read_krun_results_file(file_)
+            except IOError:
+                print("WARNING: Missing instrumenataion data for: %s:%s:%s" % \
```
Comment (ltratt) [inline bin/plot_krun_results]: Typo "instrumenataion".

**Comment by snim2** [comment]
Other than the typo this seems OK to me.

**Comment by vext01** [comment]
Squash?

**Comment by snim2** [comment]
Yup!

**Comment by vext01** [comment]
splat

**Comment by vext01** [comment]
test mattermost notify, ignore

**Review by snim2 — APPROVED** [review]

**Pull request merged**